CHANGELOG.md

```diff
@@ -130,14 +130,14 @@ Reminder: `loco --version` will give you the current Loco framework which your a
 * Added: added support in model generator for many-to-many link table generation via `loco generate model --link`
 * Docs: added Migration section, added relations documentation 1:M, M:M
 * Adding .devcontainer to starter projects [https://github.com/loco-rs/loco/issues/170](https://github.com/loco-rs/loco/issues/170)
-* **Braking changes**: Adding `Hooks::boot` application. Migration steps:
+* **Breaking changes**: Adding `Hooks::boot` application. Migration steps:
     ```rust
     // Load boot::{create_app, BootResult, StartMode} from loco_rs lib
     // Load migration: use migration::Migrator; Only when using DB
     // Adding boot hook with the following code
     impl Hooks for App {
       ...
-      async fn boot(mode: StartMode, environment: &str) -> Result<BootResult> {
+      async fn boot(mode: StartMode, environment: &Environment) -> Result<BootResult> {
```

Comment:
Why was this changed made? you change docs on version `v0.1.8, with upcoming changes to the Environment variable in the next release.

If someone wishes to upgrade from v0.1.7 to v0.1.8, what considerations should they be aware of?"